release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -565,6 +565,13 @@ You can now create DNS records by using the Red Hat External DNS Operator on clo
 
 For more information, see xref:../networking/external_dns_operator/nw-installing-external-dns-operator.adoc#nw-installing-external-dns-operator[Installing the External DNS Operator].
 
+[id="ocp-4-10-ingress-controller-endpoint-publishing-strategies"]
+==== Mutable Ingress Controller endpoint publishing strategy
+
+Cluster administrators can now configure the Ingress Controller endpoint publishing strategy in {product-title}.
```

Comment:
The specific change in OpenShift 4.10 is that the load-balancer scope can be changed between "Internal" and "External".  Some other fields were already mutable, and other fields are still not mutable.  https://github.com/openshift/openshift-docs/pull/39451#discussion_r804251703 has details.  Maybe it isn't necessary to repeat all that here as you have the cross-reference, but I'd suggest that the release note specifically mention that load-balancer scope is newly mutable (as opposed to _all_ endpoint publishing strategy settings).